src/utils/device/dehydration.ts

```diff
@@ -40,10 +41,11 @@ async function deviceDehydrationEnabled(crypto: CryptoApi | undefined): Promise<
  * @param createNewKey: force a new dehydration key to be created, even if one
  *   already exists.  This is used when we reset secret storage.
  */
-export async function initialiseDehydration(createNewKey: boolean = false): Promise<void> {
-    const crypto = MatrixClientPeg.safeGet().getCrypto();
-    if (await deviceDehydrationEnabled(crypto)) {
+export async function initialiseDehydration(opts: StartDehydrationOpts = {}, client?: MatrixClient): Promise<void> {
```

Comment:
@andybalaam @uhoreg please can we make sure that we update the *documentation* when we update the signature of a public function?

Reply:
(fix in https://github.com/element-hq/element-web/pull/29261)